### DISCUSSION TITLE
[@types/validator] Missing isTaxID definitions

### DISCUSSION THREAD

**Discussion by wejesuss** [DefinitelyTyped/DefinitelyTyped#59419] [Issues with a @types package] [answered]
Hello, I'm using `validator` module and I noticed the types definition for `isTaxID` function was missing.

I'm using `validator ^13.7.0` and `@types/validator ^13.7.1` because the definitions are missing I can't use the function without some tricks.

Module repository: [validator](https://github.com/validatorjs/validator.js)
Function implementation: [isTaxID](https://github.com/validatorjs/validator.js/blob/master/src/lib/isTaxID.js)
On [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/validator/lib) there is no such file as `isTaxID.d.ts`

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
fyi: #59426

**Reply by wejesuss** [reply]
Thank you for your pr recently, I just think you accidentally forgot to delete some unnecessary file at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8103d4f36f5531dd80f3bb7f2133bcb1609c0e4d/types/validadtor

Was this on purpose?

**Comment by peterblazejewicz** [comment]
Oh, yes, it's me. I've committed wrong file name (which make a difference in git), this was probably added in a rush to correct filename via push. That was unintended change.
Fixing this in #59419

**Reply by wejesuss** [reply]
Ok, thank you again for your changes 👍
